### DISCUSSION TITLE
Status of Governing Board

### DISCUSSION THREAD

**Discussion by modir** [laminas/technical-steering-committee#171] [Q&A]
On the [webpage about the foundation](https://getlaminas.org/about/foundation) is the following written:

> Laminas is currently governed by a [Technical Charter](https://getlaminas.org/about/tsc) under the Linux Foundation. We are also working towards a funded project model, which will entail an additional Governing Board to manage marketing the project, funding development, developing partnerships, and more.

What is the status of this Governing Board? It would be great if the whole community is informed about the current status. I am asking because I see here a lack of "marketing". I think we need more people working for this project, especially those who work on other things that are note code related. But the structure is not given for this.

**Comment by froschdesign** [comment]
@modir 
I can only speculate at this point, but perhaps the reason can be found here: https://crowdfunding.lfx.linuxfoundation.org/projects/laminas-project

**Comment by modir** [comment]
Well, if the financial situation is the problem then it would be even more important to send out a note from time to time. Because only then people and companies feel motivated to pay.

At the moment there is always the attitude "the information is already somewhere on the internet we don't need to do anything actively". And I think this is wrong.

I personally would e.g. add this link to all welcome emails sent out by discourse so that every new member knows about it. And once per year an email from either the TCS or Governing Board needs to go out to all members talking about the current status and then at the end remind the people about this link. Two small things that would make a difference.

**Reply by froschdesign** [reply]
> …add this link to all welcome emails sent out by discourse so that every new member knows about it.

Done!

> …all members…

Which members do you mean?

**Comment by modir** [comment]
> > …add this link to all welcome emails sent out by discourse so that every new member knows about it.
> 
> Done!

Wow. Very fast. Cool.

> > …all members…
> 
> Which members do you mean?

With "members" I mean the users of Discourse.
